### PR TITLE
Build engine on teamcity

### PR DESCRIPTION
**Pull request by davidfurey**

### CODE REVIEW

File: engine/deploy.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
+  "defaultStacks": [ "recommend" ],
```
Comment (agulowaty) [inline engine/deploy.json]: `recommendations` ?

**Comment by agulowaty** [comment]
:+1:

File: engine/team-city-build.sh [hunk 2]
```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+set -e
+
+script="$(cd "${0%/*}" 2>/dev/null; echo "$PWD"/"${0##*/}")"
```
Comment (agulowaty) [inline engine/team-city-build.sh:3]: bash convention is to use UPPER_CASE variable names

File: engine/team-city-build.sh [hunk 3]
```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+set -e
```
Comment (agulowaty) [inline engine/team-city-build.sh]: could be moved to she-bang probably

File: engine/team-city-build.sh (hunk 2 above)
Comment (davidfurey) [inline engine/team-city-build.sh:3]: Maybe true, but it makes the code very shouty and hard to read.  I will change it to uppercase if you have a strong preference

Comment (agulowaty) [inline engine/team-city-build.sh:3]: http://stackoverflow.com/a/673940/4877698 says you're right - lower case for script-local vars

**Comment by agulowaty** [comment]
:+1:

**Pull request merged**